components/Community/CommunityCard.jsx

```diff
@@ -1,19 +1,36 @@
 import React from "react";
 import { ProfileCard } from "../UI";
 
-const CommunityCard = ({ image, name, description}) => {
+const CommunityCard = ({ image, name, description, commitCount }) => {
+  // Custom label for the username "anmode"
+  const contributorLabel = name.toLowerCase() === "anmode" ? "Project Admin" : "Contributor";
+
+  // Function to generate the URL for the GitHub commits page for the contributor
+  const getCommitsURL = (username) => {
+    return `https://github.com/anmode/grabtern-frontend/commits?author=${username}`;
```

Comment:
it will give me how many commits? isn't?

Reply:
Yes